chart/assets/operations/addons.yaml

```
@@ -1,13 +1,40 @@
 - type: remove
   path: /addons/name=bpm
 
+# The loggr forwarder agent needs a special env var with tags that identify the job
 - type: replace
-  path: /addons/name=loggregator_agent/jobs/name=loggregator_agent/properties/quarks?/envs?
+  path: /addons/name=forwarder_agent/jobs/name=loggr-forwarder-agent/properties?/quarks/envs?
   value:
   - name: AGENT_IP
     valueFrom:
       fieldRef:
-        apiVersion: v1
         fieldPath: status.podIP
   - name: AGENT_INDEX
-    value: "0"
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.labels['statefulset.kubernetes.io/pod-name']
```

```suggestion
        fieldPath: metadata.name
```
Sorry, I accidentally deleted this comment when I meant to delete something else.